### PR TITLE
Slamdunk

### PR DESCRIPTION
**Pull request by t-neumann**
* [ x ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ x ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ x ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by t-neumann** [comment]
@bgruening Do I really need to run this? Or what's happening here?

```
#!/bin/bash -eo pipefail
.circleci/setup.sh
From ssh://github.com/bioconda/bioconda-recipes
 * [new branch]          master     -> __upstream__XN3HZ7ICTM/master
Your bioconda-recipes CI configuration is out of date.
Please update it to the latest version of the upstream master branch.
You can do this, e.g., by running:
  git fetch https://github.com/bioconda/bioconda-recipes.git master
  git merge FETCH_HEAD
Exited with code 1
```

**Comment by druvus** [comment]
@t-neumann your branch is outdated compared to master so please follow that instruction to fix it.

File: recipes/slamdunk/meta.yaml [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-  url: "https://github.com/t-neumann/slamdunk/archive/v0.3.3.tar.gz"
+  url: "https://github.com/t-neumann/slamdunk/archive/v{{ version }}.tar.gz"
 
 build:
   entry_points:
     - slamdunk=slamdunk.slamdunk:run
     - alleyoop=slamdunk.alleyoop:run
     - splash=slamdunk.splash:run
-  number: 0
+  number: 1
```
Comment (druvus) [inline recipes/slamdunk/meta.yaml]: This should be 0 for new versions.

File: recipes/slamdunk/meta.yaml [hunk 2]
```diff
@@ -40,9 +39,10 @@ requirements:
     - ncurses
     - zlib
     - bzip2
-    - curl
+    - curl >7.59
     - xz
     - libdeflate
+    - openssl=1.0
```
Comment (druvus) [inline recipes/slamdunk/meta.yaml]: Is this really needed? Or should some dependencies be rebuilt to handle current openssl version?N

File: recipes/slamdunk/meta.yaml (hunk 1 above)
Comment (t-neumann) [reply recipes/slamdunk/meta.yaml]: It's not a new version, but a new build due to the broken openssl dependency.

File: recipes/slamdunk/meta.yaml (hunk 2 above)
Comment (t-neumann) [reply recipes/slamdunk/meta.yaml]: I will fix this in the next release, but until then this is the quickfix to keep a viable version out.

**Pull request closed without merging**